draft-irtf-qirg-principles-09-rc.xml

```diff
@@ -395,14 +425,19 @@
        <t>What is so interesting about this two-qubit gate operation? The final
        state is *entangled*. There is no possible way of representing that
        quantum state as a product of two individual qubits; they are no longer
-       independent and the behaviour of either qubit cannot be fully described
-       without accounting for the other qubit. The states of the two individual
-       qubits are now correlated beyond what is possible to achieve
-       classically. Neither qubit is in a definite |0> or |1> state, but if we
-       perform a measurement on either one, the outcome of the partner qubit
-       will *always* yield the exact same outcome. The final state, whether
-       it's |00> or |11>, is fundamentally random as before, but the states of
-       the two qubits following a measurement will always be identical.</t>
+       independent. That is, it is not possible to describe the quantum state
+       of either of the individual qubits in a way that is independent of the
+       other qubit. Only the quantum state of the system that consists of both
+       qubits provides a physically complete description of the two-qubit
+       system. The states of the two individual qubits are now correlated
+       beyond what is possible to achieve classically. Neither qubit is in a
+       definite |0> or |1> state, but if we perform a measurement on either
+       one, the outcome of the partner qubit will *always* yield the exact same
+       outcome. The final state, whether it's |00> or |11>, is fundamentally
+       random as before, but the states of the two qubits following a
+       measurement will always be identical. One can think of this as flipping
+       two coins, but the coins always both land on "heads" or both land on
+       "tails" together. Something that we know is impossible classically.</t>
```

Comment:
Nice!  This is the best lay-person description of entanglement that I have heard thus far!